swagger.yaml

```diff
@@ -188,6 +292,291 @@ components:
+    ValidatorsRequest:
+      type: "object"
+      properties:
+        from:
+          type: string
+          format: date
+          example: "2019-05-17"
+          description: |
+            A starting day for which need to retrieve rewards.
+            This date should be always in UTC time zone.
+        to:
+          type: string
+          format: date
+          example: "2019-05-17"
+          description: |
+            An ending day for which need to retrieve rewards.
+            By default, assumes it is current day.
+            This date should be always in UTC time zone.
+        validators:
+          type: array
+          items:
+            type: "string"
+            description: |
+              Validator signing key, or id object
+          description: |
+            Array of indices or public signing key,
+            if empty returns all validators
+        aggregate_days:
+          type: integer
+          description: |
+            Number of daily statistics aggregated in a single entry.
+            The aggregation works as follows:
+              - for percentage values, shows avg()
+              - for gwei or count values, shows sum()
+            By default, shows single day aggregations.
+        wallets:
+          type: array
+          items:
+            type: string
+            pattern: "^(0x[a-fA-F0-9]{40})$"
+            description: |
+              Optionally submit an address of a wallet,
+              so results will be filtered only to statistics related
+              to that wallet.
+        network:
+          type: string
+          enum: ["ethereum"]
+          description: "Network for which to perform statistics request"
+      required:
+        - network
+    ValidatorsResponse:
+      type: "array"
+      items:
+        type: object
+        properties:
+          network:
+            type: string
+            enum: ["ethereum"]
+            description: "Network for which to perform statistics request"
+          balance:
+            type: string
+            pattern: "^\\d+$"
+            description: Validator balance (in gwei)
+          status:
+            type: string
+            enum: ["created", "deposited", "pending", "active", "exited"]
+          slashed:
+            type: boolean
+          mev_enabled:
+            type: boolean
+          index:
+            type: integer
+            description: On-chain index of validator (if missing, validator in deposited state)
+          public_signing_key:
+            type: string
+          withdrawal_credentials:
+            type: string
+          rewards:
+            $ref: "#/components/schemas/RewardsAndEffectivenessResponse"
+          fee_recipient:
+            type: string
+        required:
+          - status
+          - balance
+          - mev_enabled
+          - rewards
+          - withdrawal_credentials
+          - public_signing_key
+          - fee_recipient
+    # internal-begin
```

Comment:
I believe we should exclude internal sections from the update

Reply:
oO, i didn't realise there was also an internal section in between- will remove

Reply:
fixed